src/utils/withSuggestions.js

```diff
@@ -25,10 +25,10 @@ function withSuggestions(WrappedComponent) {
     render() {
       const { suggestedData, field } = this.props
       return (
-        <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
Are you sure about removing this style here?
How the suggestion icon would be renderer without this?

Reply:
you're right. i'll fix this